### ISSUE TITLE
add ability to export to an SVG to a powerpoint

### ISSUE THREAD

**Issue by wfmackey** [wfmackey/dhhstheme#11]

**Comment by MattCowgill** [comment]
the code in `grattantheme` for this is needlessly elaborate, because I needed the ability to add stuff to the slide notes, which {officer} doesn't allow, so it combines {officer} and RMd

**Comment by wfmackey** [comment]
yeah I think I'll delay the slide-notes ability for another day

**Comment by MattCowgill** [comment]
Yep. Without that it’s pretty straightforward, this function does the work in grattantheme https://github.com/grattan/grattantheme/blob/711122c4a457aca13cb01cad8c0af5d8c9cd5431/R/grattan_save_pptx.R#L248
